src/connection_string.ts

```diff
@@ -521,6 +521,9 @@ export function parseOptions(
   const loggerFeatureFlag = Symbol.for('@@mdb.enableMongoLogger');
   mongoOptions[loggerFeatureFlag] = mongoOptions[loggerFeatureFlag] ?? false;
 
+  const internalLoggerConfigSymbol = Symbol.for('@@mdb.internalLoggerConfig');
```

Comment:
Is there anywhere I should declare this symbol outside of this function? I didn't declare it anywhere else simply because it is something we only intend to use for testing.